cli/ts/commands/genProofs.ts

```diff
@@ -155,8 +155,8 @@ export const genProofs = async ({
   const messageAqContractAddr = extContracts.messageAq;
   const messageAqContract = AccQueueFactory.connect(messageAqContractAddr, signer);
 
-  // Check that the state and message trees have been merged for at least the first poll
-  if (!(await pollContract.stateAqMerged()) && pollId.toString() === "0") {
```

Comment:
Can you add this change for maci-contracts proof generator?

Reply:
sure